client/lib/phone-validation/index.jsx

```diff
@@ -40,11 +40,10 @@ module.exports = function( phoneNumber ) {
 			info: 'phone_number_valid',
```

Comment:
Can we flip this check here, it checks for positive condition, but in all other places we check for negatives. What do you think?